query.go

```diff
@@ -2162,6 +2225,59 @@ func (q *Query) BufferSize(attributeNameOrDimension string) (uint64, error) {
 	return dataNumElements, nil
 }
 
+// BufferSizeNullable returns the size (in num elements) of the backing C buffer for the given nullable attribute
+func (q *Query) BufferSizeNullable(attributeNameOrDimension string) (uint64, uint64, error) {
```

Comment:
Since nullable support is only for attributes maybe you need to set attributeName or so as an argument

Reply:
Good catch, fixed